openlibrary/plugins/recaptcha/recaptcha.py

```diff
@@ -36,24 +17,17 @@ def __init__(self, public_key, private_key):
 
         self.error = None
 
-    def render(self):
-        if self.error:
-            key = self.public_key + '&error=' + self.error
-        else:
-            key = self.public_key
-        return _recaptcha_html.replace('KEY', key)
-
     def validate(self, value=None):
         i = web.input(recaptcha_challenge_field="", recaptcha_response_field="")
 
-        data = dict(
-            privatekey=self._private_key,
-            challenge=i.recaptcha_challenge_field,
-            response=i.recaptcha_response_field,
-            remoteip=web.ctx.ip)
+        privatekey=self._private_key
+        response=i.recaptcha_response_field
+        remoteip=web.ctx.ip
+
+        data = "https://www.google.com/recaptcha/api/siteverify?secret=" + privatekey + "&response=" + response + "&remoteip=" + remoteip
 
         try:
-            response = urllib2.urlopen('http://www.google.com/recaptcha/api/verify', urllib.urlencode(data), timeout=3).read()
```

Comment:
This url needs to be `https://www.google.com/api/recaptcha/siteverify` not `https://www.google.com/recaptcha/api/siteverify`